openbb_terminal/mutual_funds/mutual_fund_controller.py

```diff
@@ -552,3 +584,233 @@ def call_forecast(self, _):
             self.data,
             self.queue,
         )
+
+
+    @log_start_end(log=logger)
+    def call_country_ms(self, other_args: List[str]):
+        """Process country_ms command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="country",
+            description="Set a country for funds",
+        )
+        parser.add_argument(
+            "-n",
+            "--name",
+            type=str,
+            dest="name",
+            nargs="+",
+            help="country to select",
+            default="",
```

Comment:
Note we have a common convention across the terminal for countries where we enter `united_states` instead of `united states`. The reason is for continuity so you dont have to remember which menus are one way or the other.